server/app.ts

```diff
@@ -1,80 +1,40 @@
 import * as fs from "fs";
 import * as path from "path";
-import * as url from "url";
-import * as os from "os";
-import * as crypto from "crypto";
-import * as http from "http";
-
 import * as express from "express";
 import * as serveStatic from "serve-static";
 import * as compression from "compression";
-import * as bodyParser from "body-parser";
-import * as cookieParser from "cookie-parser";
-import * as multer from "multer";
 import * as morgan from "morgan";
 import * as passport from "passport";
-import * as passportLocal from "passport-local"
 import * as session from "express-session"
 import * as express_graphql from "express-graphql"
 import * as cors from "cors"
 import * as dotenv from "dotenv"
-import {buildSchema} from "graphql"
-
+import { buildSchema } from "graphql"
+import { GroundTruthStrategy } from "./routes/strategies"
+import { IUser, User } from "./schema";
+import { userRoutes } from "./routes/user";
 
 dotenv.config();
+
 const PORT = 3001;
-const MONGO_URL = process.env.MONGO_URL || "mongodb://admin:teamformation123@ds121599.mlab.com:21599/hackgt-team-formation";
-const UNIQUE_APP_ID = process.env.UNIQUE_APP_ID || "team-formation";
 const STATIC_ROOT = "../client";
-
-const VERSION_NUMBER = JSON.parse(fs.readFileSync(path.resolve(__dirname, "../package.json"), "utf8")).version;
-const VERSION_HASH = require("git-rev-sync").short();
 const typeDefs = fs.readFileSync(path.resolve(__dirname, "../api.graphql"), "utf8");
 
 export let app = express();
 app.use(morgan("dev"));
 app.use(compression());
 app.use('*', cors());
 
-let cookieParserInstance = cookieParser(undefined, {
-	"path": "/",
-	"maxAge": 1000 * 60 * 60 * 24 * 30 * 6, // 6 months
-	"secure": false,
-	"httpOnly": true
-} as cookieParser.CookieParseOptions);
-app.use(cookieParserInstance);
 let session_secret = process.env['SECRET'] || 'default';
```

Comment:
If session_secret isn't define, make it something random or don't start and throw an error.  Setting it to a default value could potentially create a security vulnerability if the value isn't set in a production deployment, especially since there's no warning for this